### DISCUSSION TITLE
[Closed] OpenXLA Monthly Community Meetings

### DISCUSSION THREAD

**Discussion by theadactyl** [openxla/community#10] [Announcements]
**Update 2**: We've converged on Third Tuesdays at 8am PT. 

**Update:** Please respond here instead: https://github.com/openxla/community/discussions/12

Hi folks, 

We are working to schedule the next community meeting, and would like to land on a consistent monthly schedule contributors can plan around. 

Though it's early for the US West Coast, 8AM seems to be a good option to be a good option to include East Coast and EMEA contributors. 

For APAC contributors, 4PM or 5PM may be good options, for monthly or every other month meetings depending on folks' availability. 

Please let us know which meeting times you could attend regularly.

**Comment by brnorris03** [comment]
Only single select is allowed, can you change to multi-select? Thanks!

**Comment by theadactyl** [comment]
Hmmm, apologies -- I thought this was supported in GitHub Polls but looks like it isn't . Let me look at an alternative.

**Comment by theadactyl** [comment]
Hacky take two of the survey here: https://github.com/openxla/community/discussions/12
